### PR TITLE
Release `0.28.0-rc.0`

### PR DESCRIPTION
**Pull request by neysofu**
This is the first release candidate for `v0.28.0`. `NEWS.md` has been updated with the release notes from [this](https://github.com/graphprotocol/graph-node/compare/v0.27.0...1435fceb5840c479209c54330ce0dc6b5a3c1b6e) commit range.

### CODE REVIEW

**Comment by neysofu** [comment]
On hold; waiting for https://github.com/graphprotocol/graph-node/issues/3936.

**Comment by neysofu** [comment]
Ready for review. Will cherrypick https://github.com/graphprotocol/graph-node/commit/6387465040b98f926880f069c655b8f5ea1f1b4b on top of this branch after merging with `master`.

File: NEWS.md
Comment (azf20) [inline NEWS.md:26]: Versioning

File: NEWS.md
Comment (azf20) [inline NEWS.md:21]: "available for subgraphs indexing EVM chains"

File: NEWS.md
Comment (azf20) [inline NEWS.md:47]: @lutter did we not revert this change?

**Review by azf20 — APPROVED** [review]
I think this looks good! Some small comments

File: NEWS.md
Comment (neysofu) [reply NEWS.md]: Fixed! Thanks for the catch

File: NEWS.md
Comment (neysofu) [reply NEWS.md]: 👍

**Pull request merged**
